xhprof_lib/utils/xhprof_runs.php

```diff
@@ -107,22 +107,22 @@ public static function getDbClass()
   * 
   *
 CREATE TABLE `details` (
```

Comment:
1. any reason for changing size of data in columns (e.g. `char(17)` into `char(64)` and so on?
2. any reason for random keyword case changing (e.g. `blob` to `BLOB`, but not `varchar` to `VARCHAR`)?